installscripts/cookbooks/jenkins/attributes/default.rb

```diff
@@ -25,6 +25,7 @@
 default['bbpassword'] = 'REPLACEME'
 default['sonaruser'] = 'REPLACEME'
 default['sonarpassword'] = 'REPLACEME'
+default['scenario'] = 'scenario1'
```

Comment:
As with the Terraform variables, change this to something more specific like `dockerizedJenkins true|false`

Reply:
Done